### ISSUE TITLE
Update to newer versions

### ISSUE THREAD

**Issue by AfkF24** [novucs/factions-top#131]
Hi,

Any plans of getting Factions Top updated to support 1.15.2?
Building the plugin seems to be broken too.

**Comment by SlickJava** [comment]
Yes please. I've been searching for a working block-calc factions top for 1.15.2 (compatible with FactionsUUID) for a while now... no luck at all.

**Comment by Phoenix616** [comment]
Unfortunately updating past 1.13 is a pretty large undertaking as lots of block/item material names changed and a certain amount of work as well as testing has to be put into if the goal should be to keep the update compatible with 1.8 (which lots of factions server still seem to use adamantly).

If the goal was to just get it running on 1.13+ then it shouldn't be too hard but I doubt anyone wants to maintain two separate branches but if anyone thinks it would be a better solution than not having any at all then of course they can create a PR for a quick and dirty fix for 1.13+.

Also ideally you would provide some more information regarding what actually breaks when using in  a version past 1.13 as well as for the building. (Which seems to work just fine for me)
